### PR TITLE
Devtool improvement for the flattengetters

### PR DESCRIPTION
**Pull request by DevoidCoding**
The loop should only loop if the property is a module or a submodule

### CODE REVIEW

File: src/devtool-plugin.ts [hunk 1]
```diff
@@ -39,24 +39,24 @@ function proxyStore (store: VueStore<{}, BG0, BM0, BA0>) {
...
+
+            const desc = Object.getOwnPropertyDescriptor(getters, key)
+            if (!(getters[key].__proto__ instanceof BG)) {
+                Object.defineProperty(acc, path.concat(key).join('.'), {
+                    get: () => getters[key],
+                    enumerable: true,
+                    configurable: true
+                })
+            }
+            if (getters[key].__proto__ instanceof BG)
```
Comment (ktsn) [inline src/devtool-plugin.ts]: Can we replace this line with `else` and add braces around this block?

**Comment by ktsn** [comment]
Also can we retain the indent with 2 spaces?

**Comment by DevoidCoding** [comment]
Hi ! Sorry I was on vacantion !
For sure we can, I'm gonna edit that. I don't have your tslint configuration, so mine is breaking everything, sorry for that ^^

**Comment by ktsn** [comment]
@DevoidCoding Nevermind. Thank you!

**Pull request merged**